Index: b/data/bs.cfg
===================================================================
--- a/data/bs.cfg	2016-04-24 14:27:35.935565184 +0200
+++ b/data/bs.cfg	2016-04-24 14:27:52.195564603 +0200
@@ -26,11 +26,11 @@
 # The maximum number of peer hosts in the pool, defaults to 1000.
 host_pool_capacity = 1000
 # The peer hosts cache file path, defaults to 'hosts.cache'.
-hosts_file = hosts.cache
+hosts_file = /var/lib/libbitcoin/server/hosts.cache
 # The debug log file path, defaults to 'debug.log'.
-debug_file = debug.log
+debug_file = /var/log/libbitcoin/server_debug.log
 # The error log file path, defaults to 'error.log'.
-error_file = error.log
+error_file = /var/log/libbitcoin/server_error.log
 # The advertised public address of this node, defaults to none.
 self = 0.0.0.0:0
 # A seed node for initializing the host pool, multiple entries allowed, defaults shown.
@@ -54,7 +54,7 @@
 # The history index start height, defaults to 0.
 history_start_height = 0
 # The blockchain database directory, defaults to 'blockchain'.
-database_path = blockchain
+database_path = /var/lib/libbitcoin/server/blockchain
 # A hash:height checkpoint, multiple entries allowed, defaults shown.
 checkpoint = 000000000019d6689c085ae165831e934ff763ae46a2a6c172b3f1b60a8ce26f:0
 checkpoint = 0000000069e244f73d78e8fd29ba2fd2ed618bd6fa2ee92559f542fdb26e7c1d:11111
